packages/proxy/lib/http/util/inject.ts

```diff
@@ -21,11 +21,12 @@ export function full (domain) {
   })
 }
 
-export function fullCrossOrigin (domain) {
+export function fullCrossOrigin (domain, { modifyObstructiveThirdPartyCode, modifyObstructiveCode }) {
   return getRunnerCrossOriginInjectionContents().then((contents) => {
     return oneLine`
       <script type='text/javascript'>
         document.domain = '${domain}';
+        window.cypressConfig = { modifyObstructiveThirdPartyCode: ${modifyObstructiveThirdPartyCode}, modifyObstructiveCode: ${modifyObstructiveCode} };
```

Comment:
we inject these config object into the cross origin code to allow us to know if we should patch or not prior to a spec bridge attaching.

Reply:
Wrap in a closure providing the variables instead of attaching them to window.